dashboard/src/app/projects/project-details/project-details.controller.js

```diff
@@ -59,22 +59,46 @@ export class ProjectDetailsCtrl {
   }
 
   fetchProjectDetails() {
-    this.projectService = this.cheAPI.getWorkspace().getWorkspaceAgent(this.workspaceId).getProject();
+    if (this.workspace.status === 'STARTING' || this.workspace.status === 'RUNNING') {
```

Comment:
We can use next:
    if (this.workspace.status !== 'STARTING' && this.workspace.status !== 'RUNNING') {
      this.loading = false;
      this.noWorkspaceRuntime = true;
      return;
    }
WDYT?